project/Dependencies.scala

```diff
@@ -20,6 +20,14 @@ object Dependencies {
     )
   }
 
+  val playLibs_2_8 = {
+    val version = "2.8.1"
+    Seq(
+      "com.typesafe.play" %% "play" % version % "provided",
```

Comment:
ooh, what does `provided` do?

Reply:
It indicates to the package manager that projects using this library will provide their own version of the package rather than have it download it as a transitive dependency of this library.

I think we use it here to avoid problems when Play is brought in via the SBT plugin in projects that use Panda